rift/node.py

```diff
@@ -2835,6 +2837,22 @@ def spf_add_pos_disagg_prefixes(self, node_sysid, node_cost, candidates, spf_dir
                                          pos_disagg_prefixes,
                                          is_pos_disagg=True, is_neg_disagg=False)
 
+    def spf_add_neg_disagg_prefixes(self, node_sysid, node_cost, candidates, spf_direction,
+                                    special_for_neg_disagg):
+        prefix_ties = self.ties_of_type(
+            direction=self.spf_use_tie_direction(node_sysid, spf_direction),
+            system_id=node_sysid,
+            prefix_type=common.ttypes.TIETypeType.NegativeDisaggregationPrefixTIEType)
+
+        for prefix_tie in prefix_ties:
+            neg_disagg_prefixes = prefix_tie.element.negative_disaggregation_prefixes.prefixes
+            # Force node_cost = 0 since the prefix metric is infinite so we don't need to compute a real cost
+            # TODO: Ask to Bruno
+            self.spf_add_prefixes_common(node_sysid, 0, candidates, spf_direction,
```

Comment:
@brunorijsman here we set the node cost to 0, otherwise the "infinite distance" will still increment (and it's not right)